opennms-webapp-rest/src/test/java/org/opennms/web/rest/v2/MonitoringLocationRestServiceIT.java

```diff
@@ -164,4 +177,208 @@ public void testEventOnCreationAndDeletion() throws Exception {
         // delete the one with polling packages
         sendRequest(DELETE, "/monitoringLocations/location2", 204);
     }
+
+    @Test
+    @Transactional
+    public void testDefaultLimitLocations5() throws Exception {
+        // 5 <= current default of 10, should return 5 (includes default)
+        assertEquals(5, testLocationLimit(5,DEFAULT_FLAG));
+    }
```

Comment:
Combine all these test into one. Create your locations at the beginning and test the few cases.